packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -1706,32 +1711,75 @@ export const audiusBackend = ({
     return receipt
   }
 
-  async function getAssociatedTokenAccountInfo(address: string) {
-    await waitForLibsInit()
+  async function getTokenAccount({
+    address,
+    sdk
+  }: {
+    address: PublicKey
+    sdk: AudiusSdk
+  }) {
+    const connection = sdk.services.solanaClient.connection
```

Comment:
Just gonna throw it out there: This function could just take the connection as an argument and not have to know about sdk at all...

Reply:
hm true. will leave for now.. stick to pass in sdk pattern (even tho pattern is whatevs)